Target.java

```diff
@@ -1,17 +1,63 @@
-/*
- * File: Target.java
- * Name: 
- * Section Leader: 
- * -----------------
- * This file is the starter file for the Target problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Target extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
+/*

```

Comment:
This implementation does not work. I did not see the target being drawn when I ran the code.